interactive_engine/src/common/java/src/main/java/com/alibaba/maxgraph/common/cluster/InstanceConfig.java

```diff
@@ -54,6 +54,7 @@
      */
     public static final String PEGASUS_WORKER_NUM = "pegasus.worker.num";
     public static final String PEGASUS_TIMEOUT = "pegasus.timeout";
+    public static final String GAIA_ENABLE = "gaia.enable";
```

Comment:
Probably "gaia.compiler.enabled" and `GAIA_COMPILER_ENABLED` are more suitable names. And add a comment that this is a switch to determine whether Gaia or MaxGraph's compiler is used.